bspump/ipc/datagram.py

```diff
@@ -123,8 +117,6 @@ def __init__(self, app, pipeline, id=None, config=None):
 
 			self.Socket = socket.socket(socket.AF_UNIX, socket.SOCK_DGRAM)
 			self.Socket.setblocking(False)
-			self.Socket.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
-			self.Socket.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEPORT, 1)
```

Comment:
`SO_REUSEADDR` is likely not applied to AF_UNIX at all ... I suggest to remove the setting from here.